### PR TITLE
Dynamic user pages

### PR DESCRIPTION
**Pull request by EgoriusRus**

### CODE REVIEW

**Review by kezlya — APPROVED** [review]
lgtm

File: pages/users/index.vue [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+<script>
+import UserProfile from '@/components/UserProfile'
+
+export default {
+  name: 'User',
+  components: {
+    UserProfile
+  },
+  data: () => ({
+    username: ''
```
Comment (Klimonov) [inline pages/users/index.vue]: please use the recommended version of date definition `data() {`

File: components/UserProfile.vue [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+      return '/img/user_background.png'
+    },
+    getUserSocials() {
+      if (this.getUser && this.getUser.socials && this.getUser.socials.length) {
+        return this.getUser.socials
+      }
+    }
+  },
+  async mounted() {
+    await this.loadGames()
```
Comment (Klimonov) [inline components/UserProfile.vue]: I believe you no need to use await here. You ask await for what ?

File: components/UserProfile.vue [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+  },
+  created() {
+    this.us = new User()
+    this.us.getUserData(this.username).then((result) => {
```
Comment (Klimonov) [inline components/UserProfile.vue]: please use async await instead of unfashionable callbacks

**Review by Klimonov — APPROVED** [review]
LGTM

**Pull request merged**